src/main/java/br/com/deveficiente/nossomercadolivreapi/produto/ProdutosController.java

```diff
@@ -0,0 +1,48 @@
+package br.com.deveficiente.nossomercadolivreapi.produto;
+
+import br.com.deveficiente.nossomercadolivreapi.shared.infra.Uploader;
+import br.com.deveficiente.nossomercadolivreapi.usuario.Usuario;
+import br.com.deveficiente.nossomercadolivreapi.usuario.UsuarioRepository;
+import org.reflections.Reflections;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.WebDataBinder;
+import org.springframework.web.bind.annotation.InitBinder;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping("/api/produtos")
+public class ProdutosController {
+
+    @Autowired
+    private UsuarioRepository usuarioRepository;
+
+    @Autowired
+    private Uploader uploader;
+
+    @Autowired
+    private ProdutoRepository produtoRepository;
+
+    @PostMapping
+    public void cria(@Valid ProdutoRequest produtoRequest) {
+
+        String email = "usuario@email.com.br";
+        Usuario usuario = usuarioRepository.findByLogin(email).get();
+
+        verificaSeUsuarioJaTemProdutoCadastrado(usuario);
```

Comment:
Essa restrição aqui surgiu de onde? To perguntando pq não achei no cartão e aí fiquei em dúvida :).

Reply:
Acho que entendi errado então hehe, foi no texto abaixo:

"Aqui a gente vai permitir o cadastro de um produto por um usuário logado."

Reply:
Vi aqui! Foi maaaaal! corrigi :)

Reply:
Já removi também...hehe
Valeu.